From 3872830620e4ad398d7926d81079744da09abcfa Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Fri, 18 Jan 2019 14:18:53 +0000
Subject: [PATCH 2/4] FinishActivity: Remove useless call to
 LineageHardwareManager

 * This is properly handled in fw/b once the setting is changed

Change-Id: I98acd00204b3f741e34069fbd9ba016fbc15c8dc
---
 src/org/lineageos/setupwizard/FinishActivity.java | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/src/org/lineageos/setupwizard/FinishActivity.java b/src/org/lineageos/setupwizard/FinishActivity.java
index 858d3bb..c02ed0a 100644
--- a/src/org/lineageos/setupwizard/FinishActivity.java
+++ b/src/org/lineageos/setupwizard/FinishActivity.java
@@ -1,6 +1,6 @@
 /*
  * Copyright (C) 2016 The CyanogenMod Project
- * Copyright (C) 2017-2018 The LineageOS Project
+ * Copyright (C) 2017-2019 The LineageOS Project
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -44,7 +44,6 @@ import com.android.setupwizardlib.util.WizardManagerHelper;
 
 import org.lineageos.setupwizard.util.EnableAccessibilityController;
 
-import lineageos.hardware.LineageHardwareManager;
 import lineageos.providers.LineageSettings;
 
 public class FinishActivity extends BaseSetupWizardActivity {
@@ -213,8 +212,6 @@ public class FinishActivity extends BaseSetupWizardActivity {
 
         LineageSettings.System.putIntForUser(context.getContentResolver(),
                 LineageSettings.System.FORCE_SHOW_NAVBAR, enabled ? 1 : 0, UserHandle.USER_CURRENT);
-        LineageHardwareManager hardware = LineageHardwareManager.getInstance(context);
-        hardware.set(LineageHardwareManager.FEATURE_KEY_DISABLE, enabled);
 
         /* Save/restore button timeouts to disable them in softkey mode */
         if (enabled) {
-- 
2.17.1

